src/studio/nuxeo-studio/nuxeo-studio-designer-git-access.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Nuxeo Studio - Designer Git Access
+description: Learn how to use direct access to the underlying Git project for Studio Designer.
+review:
+    comment: ''
+    date: ''
+    status: ok
+toc: false
+tree_item_index: 700
+---
+
+## About Studio Designer Git Access
+When configuring Web UI, Studio Designer allows to edit the generated files directly and add your custom HTML / Javascript code. This allows developers to do advanced customization.
+
+While this is great, the browser is not a good place to do anything beyond simple tweaks and as a developer, we understand that you want to use the tools you are familiar with. Nuxeo Studio Designer Git access aims at solving this problem: you can clone your project, edit it locally with your favorite tools and benefit from advanced Git features, then push your changes back in Nuxeo Studio.
+
+## Who Can Use It?
+Studio Designer Git Access is available to any customer benefiting from the branch management feature.
+To activate it, simply request it through a support ticket by indicating:
+- Who should be given access
+- Which access these people should obtain (read or write)
+- On which project(s)
+
+## Feature Scope and Restrictions
+
+ Nuxeo Studio Designer Git access has been designed to allow for productivity, while keeping sane restrictions for security and ease of use reasons.
+
+### What's possible
+- Access to Nuxeo Studio Git repositories through HTTPS protocol
+- Launch git commands on your repositories, with some restrictions
+- Automatic refresh of the Studio project cache when git commands have been launched
```

Comment:
What is the project cache?

Reply:
Something you removed in the dev process :-) bad copy pasta from the beta version doc.

Reply:
Okay. As it is an implementation detail in any case, it shouldn't be in the user doc, just technical IMHO